tasklist/client/src/Processes/History/index.tsx

```diff
@@ -30,21 +29,19 @@ const History: React.FC = () => {
 
   return (
     <div className={styles.container}>
-      <Label $color="secondary" className={styles.header}>
-        History
-      </Label>
+      <span className={styles.header}>History</span>
```

Comment:
Can't we create global utility classes to replace the components? `.labelFont`, `.bodyCompactFont`, etc

Reply:
We could use the `type-classes` mixin.

https://carbondesignsystem.com/elements/typography/code

This will give us a set of helper classes but we would end up having styles split between JSX and Stylesheets. I would prefer to keep as much as possible in the stylesheets.

Do we want to create a class per-function (for example, `title`, `subtitle`) or do we want something more like bootstrap or tailwind where a CSS class adds a particular function (like `mx-4`, `py-4`, `align-center`)?

Reply:
Also can't delete the file since other tokens are still in use. I can get rid of Label.

Reply:
[I was thinking more along the lines creating a class for each font token](https://carbondesignsystem.com/elements/typography/type-sets). That's what I meant by `.labelFont`, maybe `.labelFontToken` is better.

We could also create classes for font colors too with `.textPrimary`, etc